WBooks/Extensions/UIColorExtension.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UIColorExtension.swift
+//  WBooks
+//
+//  Created by Ariel Cid on 3/16/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIColor {
+    
+    static func fontDarkGray() -> UIColor {
```

Comment:
I prefer use a more general name in this functions, a good practice could be use this next page to name the color:
http://chir.ag/projects/name-that-color/

Reply:
Dark Gray is a real UIColor, but the other one should be renamed. I'll be using "polar" for that particular shade of light blue.

Reply:
ok nice but I mean the `font` should be removed so we can use it in another places like backgroundColor or whatever